seed: 3

map_length: 30.0
map_width: 15.0

max_obstacle_length: 2.0
max_obstacle_width: 2.0
max_obstacle_height: 1.0

min_obstacle_length: 0.5
min_obstacle_width: 0.5
min_obstacle_height: 0.1
max_obstacle_roughness: 0.1

min_slope: 50

resolution: 0.25
num_obstacles: 5
